site/app/templates/forum/ShowCategories.twig

```diff
@@ -0,0 +1,125 @@
+{% if popup == false %}
```

Comment:
Since we are doing away with the popup, we should remove these if checks that supported both.

Reply:
Sure!